src/NzbDrone.Core/Download/Clients/QBittorrent/QBittorrentStalledPolicy.cs

```diff
@@ -0,0 +1,68 @@
+using System;
+
+namespace NzbDrone.Core.Download.Clients.QBittorrent
+{
+    public interface IQBittorrentStalledPolicy
+    {
+        void Apply(QBittorrentTorrent torrent, DownloadClientItem ditem, QBittorrentSettings settings);
```

Comment:
Would rather make this a constructor provided service to the policy, but the downloader's base class owns the property and I didn't find a way to wire a common instance from outside.